<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtProcessInfoToolsClass</class>
 <widget class="QMainWindow" name="QtProcessInfoToolsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtProcessInfoTools</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="TableProcessInfo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>711</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="LabRGB">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="LabRGBState">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>3</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>驱动状态:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="OpenStart">
    <property name="geometry">
     <rect>
      <x>268</x>
      <y>127</y>
      <width>120</width>
      <height>94</height>
     </rect>
    </property>
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionRunDriver"/>
    <addaction name="actionStop"/>
   </widget>
   <addaction name="OpenStart"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRunDriver">
   <property name="text">
    <string>运行驱动</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>关闭驱动</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtProcessInfoTools.qrc"/>
 </resources>
 <connections/>
</ui>
